api/src/backend/api/rls.py

```diff
@@ -59,11 +58,11 @@ class RowLevelSecurityConstraint(models.BaseConstraint):
     drop_sql_query = """
         ALTER TABLE %(table_name)s NO FORCE ROW LEVEL SECURITY;
         ALTER TABLE %(table_name)s DISABLE ROW LEVEL SECURITY;
-        REVOKE ALL ON TABLE %(table_name) TO %(db_user)s;
+        REVOKE ALL ON TABLE %(table_name)s FROM %(db_user)s;
```

Comment:
This change looks so important, can you give me more information? Was this working before this change?

Reply:
It wasn't working, it's part of the fix for the revert migrations. We haven't used this until now, that I split the migrations and tested if we could revert them.

Reply:
Understood, thank you